cmake/packages_check.cmake

```
@@ -68,3 +68,26 @@ elseif(TARGET_CPU STREQUAL "aarch64")
     set(libvpx_target "arm64-win64-gcc")
     set(mpv_gl "-Dgl=disabled -Degl-angle=disabled")
 endif()
+
+set(cmake_conf_args
+	-G Ninja
+	-DCMAKE_BUILD_TYPE=Release
+	-DCMAKE_TOOLCHAIN_FILE=${TOOLCHAIN_FILE}
+	-DCMAKE_INSTALL_PREFIX=${MINGW_INSTALL_PREFIX}
+	-DCMAKE_FIND_ROOT_PATH=${MINGW_INSTALL_PREFIX}
+	-DBUILD_SHARED_LIBS=OFF
+    -DBUILD_TESTING=OFF
+)
+set(meson_conf_args
+    --prefix=${MINGW_INSTALL_PREFIX}
+    --libdir=${MINGW_INSTALL_PREFIX}/lib
+    --cross-file=${MESON_CROSS}
+    --buildtype=release
+    --default-library=static
+)
```

```suggestion
set(meson_conf_args
    --cross-file=${MESON_CROSS}
)
```
They should be moved into the cross-file
```
[built-in options]
prefix = '@MINGW_INSTALL_PREFIX@'
libdir = '@MINGW_INSTALL_PREFIX@/lib'
buildtype = 'release'
default_library = 'static'
pkgconfig.relocatable = true
```

---

That doesn't seem to make a difference? Is there a better reason?

---

> That doesn't seem to make a difference? Is there a better reason?

meson.build_options() doesn't include the options defined in cross-file.